internal/interfaces/grpc/handler/operator.go

```diff
@@ -146,6 +146,28 @@ func (o operatorHandler) ReportMarketFee(
 	return o.reportMarketFee(ctx, req)
 }
 
+func (o operatorHandler) ReloadUtxos(
+	ctx context.Context,
+	rew *pb.ReloadUtxosRequest,
+) (*pb.ReloadUtxosReply, error) {
+	res, err := o.dbManager.RunTransaction(
```

Comment:
so it's not a real conflict, rather than "db is busy".  In this scenario seems to me that is acceptable to have a **stale read** if the operator is reloading utxos set in the same moment is deriving a new address or a trade is deriving a new change address, is ok to me to reload the "old" list of address only.

but due snapshot isolation of badger it seems to me that this is not possible right? to "force" a stale read I mean.

Reply:
> but due snapshot isolation of badger it seems to me that this is not possible right? to "force" a stale read I mean.

i dont think it is possible to read stale data

> The conflict that can arise here just if it fails to acquire the lock on vault repo to retieve the Vault right?

I think that it transaction conflict will be thrown if vault read happen before some other tx updates vault.

Reply:
gotcha 👍  although this seems a read-only tx, should not interfer that much https://dgraph.io/docs/badger/get-started/#read-only-transactions